torchvision/models/detection/faster_rcnn.py

```diff
@@ -298,6 +304,40 @@ def forward(self, x):
         return x
 
 
+class FastRCNNHeads(nn.Sequential):
```

Comment:
nit: Naming is hard. The first version of Fast R-CNN didn't use FPN, and the head [was actually the last stage of a ResNet](https://github.com/facebookresearch/maskrcnn-benchmark/blob/57eec25b75144d9fb1a6857f32553e1574177daf/maskrcnn_benchmark/modeling/roi_heads/box_head/roi_box_feature_extractors.py#L14). When FPN was introduced, they made the head lighter and thus the name TwoMLP.

I would be ok leaving the name as `FastRNNHeads` in here, but maybe a naming a [bit more specific like the one used in Detectron2](https://github.com/facebookresearch/detectron2/blob/6886f85baee349556749680ae8c85cdba1782d8e/detectron2/modeling/roi_heads/box_head.py#L26) might be preferable?

On an unrelated note, they actually fused both options together by allowing convs followed by linear layers

Reply:
Happy to make the naming change to `FastRCNNConvFCHead`. I used the specific one to keep it aligned with the equivalent `MaskRCNNHeads` class.

> On an unrelated note, they actually fused both options together by allowing convs followed by linear layers

Good idea. I can extend it with `conv_layers` and `fc_layers` params on the head constructor to support both. Would what work?

Reply:
Yes, that would work and could generalize TwoMLPHead, so that it could just inherit from it instead (for BC)

Reply:
We discussed offline with @fmassa the possibility of making `TwoMLPHead` inherit from `FastRCNNConvFCHead`. We decided not to do this because in order to keep things BC we would have to add another lengthy `_load_from_state_dict` implementation which adds more code than the inheritance saves. Thus we decided to keep `TwoMLPHead` as-is.